jupyter-notebooks/Subscriptions-API/Subscriptions_API_Quickstart.ipynb

```diff
@@ -65,164 +140,248 @@
+   "id": "f3986691-4942-464d-8b7e-0e2dabc8f20b",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from planet import Session\n",
-    "from planet.clients.subscriptions import SubscriptionsClient"
+    "existing_subscriptions = list(pl.subscriptions.list_subscriptions(status=['running', 'pending', 'completed'], limit=5))"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "2cef1b0c",
+   "id": "bb05d7f0-d0a7-4b0a-bce0-ba1ac41d5ea8",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# We want to initiate and authenticate into the Client, and then list up to 5 of our recent running or pending subscriptions\n",
-    "async with Session(auth=auth) as sess:\n",
-    "    cl = SubscriptionsClient(sess)\n",
-    "    subscriptions = cl.list_subscriptions(limit=5, status=['running', 'pending', 'completed'])\n",
-    "    sub_list = [i async for i in subscriptions]"
+    "existing_subscriptions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c5fbfb96-11f9-4cc4-bedf-7b80c778c1bc",
+   "metadata": {},
+   "source": [
+    "_______"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fffa6e0e",
+   "metadata": {},
+   "source": [
+    "## Creating a Subscription"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f7010fe-c520-45a0-b89a-0589c5496e5f",
+   "metadata": {},
+   "source": [
+    "**WARNING: Creating a Request below will consume quota! Be sure about the filter params so you can create a useful subscription. The variables below are meant as placeholders.**\n",
+    "\n",
+    "In order to create a subscription using the Python Client, we must provide a subscription description dictionary. This will include a `name`, `source` block, along with a `delivery` or a `hosting` block with a specified cloud storage location or cloud hosting platform, respectively. Subscriptions also support select tools. We will accomplish this all using `build_request()`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99d2453c-2e6b-450b-821e-5ed083346cb1",
+   "metadata": {},
+   "source": [
+    "#### Cloud Delivery Parameters:\n",
+    "\n",
+    " Your delivery or hosting parameters may be specified here. Below is a template for delivery to a Google Cloud Storage bucket. Note that you will need a serivce account with create, delete, and get persmissions and it is recommended to store your GCS credentials as an environment variable. See the [Google Cloud documentation](https://cloud.google.com/iam/docs/keys-create-delete#iam-service-account-keys-create-console) on how to create a service account key with the appropriate permissions. It is recommended to set the GCS credentials as an environment variable; more information is available [here](https://docs.planet.com/develop/apis/subscriptions/delivery/#google-cloud-storage). \n",
+    " <br> <br>\n",
+    " Other options may be found with `subscription_request.amazon_s3`, `subscription_request.oracle_cloud_storage`, etc...\n"
```

Comment:
You already mention the other supported destinations above, I don't think it's necessary to call out the specific methods - thats what the docs are for.